<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>814</width>
    <height>486</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QGridLayout" name="gridLayout">
    <item row="0" column="0">
     <widget class="QListWidget" name="listWidget">
      <property name="maximumSize">
       <size>
        <width>300</width>
        <height>16777215</height>
       </size>
      </property>
     </widget>
    </item>
    <item row="0" column="1">
     <widget class="QTextBrowser" name="textBrowser"/>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>814</width>
     <height>20</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>File</string>
    </property>
    <addaction name="actionInput_Alignment"/>
    <addaction name="actionFetch_From_PFAM"/>
    <addaction name="actionExport_Alignment"/>
   </widget>
   <widget class="QMenu" name="menuMethod">
    <property name="title">
     <string>Methods</string>
    </property>
    <addaction name="actionFilter_Alignment"/>
    <addaction name="actionCalculate_Minimum_Sub_Alignment_Size"/>
    <addaction name="actionCalculate_Correlation_Scores"/>
    <addaction name="actionDecompose_Network"/>
    <addaction name="actionAnalyze_Conservation_Sets"/>
   </widget>
   <widget class="QMenu" name="menuHelp">
    <property name="title">
     <string>Help</string>
    </property>
    <addaction name="actionAbout"/>
   </widget>
   <widget class="QMenu" name="menuShow">
    <property name="title">
     <string>Show</string>
    </property>
   </widget>
   <addaction name="menuFile"/>
   <addaction name="menuMethod"/>
   <addaction name="menuShow"/>
   <addaction name="menuHelp"/>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>true</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <action name="actionInput_Alignment">
   <property name="text">
    <string>Input Alignment</string>
   </property>
  </action>
  <action name="actionFetch_From_PFAM">
   <property name="text">
    <string>Fetch From PFAM</string>
   </property>
  </action>
  <action name="actionExport_Alignment">
   <property name="text">
    <string>Export Alignment</string>
   </property>
  </action>
  <action name="actionFilter_Alignment">
   <property name="text">
    <string>Filter Alignment</string>
   </property>
  </action>
  <action name="actionCalculate_Minimum_Sub_Alignment_Size">
   <property name="text">
    <string>Calculate Minimum Sub-Alignment Size</string>
   </property>
  </action>
  <action name="actionCalculate_Correlation_Scores">
   <property name="text">
    <string>Calculate Correlation Scores</string>
   </property>
  </action>
  <action name="actionDecompose_Network">
   <property name="text">
    <string>Decompose Network</string>
   </property>
  </action>
  <action name="actionAnalyze_Conservation_Sets">
   <property name="text">
    <string>Analyze Conservation Sets</string>
   </property>
  </action>
  <action name="actionAbout">
   <property name="text">
    <string>About</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
